### DISCUSSION TITLE
The latest commits messed up wifi completely on AX3600

### DISCUSSION THREAD

**Discussion by felipejfc** [Boos4721/openwrt#55] [Bugs/问题反馈]
I built 6b5810b and flashed in my router, however wifi got unstable and it's leaking memory like crazy, restarting every 6 hours more or less (even with ipv6 disabled)

I noticed also in low-latency streaming applications, such as GeForce experience, the 5G network is very unstable when compared to the version I built using commit 9979dc

**Comment by Boos4721** [comment]
kk, just revert 322713377e203c86152b3bd2f25bb0ea2adfa86b and dd4504b823f57692644d0e2f239ea9dcf449aa8a. or use stable branch.

**Reply by felipejfc** [reply]
thanks @Boos4721 I didn't realize we had a stable branch!

btw the build is broken for it right now, there's a conflict in the patch ```700-ipq8074-overclock-cpu-2.2ghz.patch```

**Reply by Boos4721** [reply]
now, fixed.
